src/main/java/com/google/api/codegen/py/PythonSphinxCommentFixer.java

```diff
@@ -22,6 +22,8 @@
 
   /** Returns a Sphinx-formatted comment string. */
   public static String sphinxify(String comment) {
+    // escape '$' in the comment first
```

Comment:
It is unclear to me why '$' needs to be replaced. Can you bring a URL to explain what '$' sign means in Sphinx?